lunes_cms/cms/admins/document_admin.py

```diff
@@ -28,6 +28,7 @@ class DocumentAdmin(admin.ModelAdmin):
     readonly_fields = ("created_by",)
```

Comment:
This should be integrated now. I hope I did it correctly.

About the hide/show: I tried to prototype the toggle. However I wasn't sure where to put the JS code. Do you think it's better to do it via a widget like for the ManyToManyOverlay or do you think it's better to load the file from inside the base.html?

Reply:
Yes, I think it makes sense to outsource this into a separate file since it is only needed in the document form and not on all other pages.
But you don't need a widget for this, you can just add this to the media class of the document admin:
https://github.com/digitalfabrik/lunes-cms/blob/6ac4c96fe391c77ca4429a0127e16e0a06e67551/lunes_cms/cms/admins/document_admin.py#L224

Reply:
Also, I'd suggest to create a new css file `document_form.css` with the following content:
```
.fieldBox.field-plural, .fieldBox.field-word {
  flex-grow: 1 !important;
}
```
and add this to the meta class as well:
```
css = {"all": ("css/document_form.css",)}
```
(to make the word input fields a bit wider)

Reply:
Both should be resolved. If I put the jQuery code inside the new file, I still get an error saying "Uncaught TypeError: $ is not a function". 
My thought process here was:
- maybe jQuery doesn't get loaded. I tested like [described here](https://stackoverflow.com/questions/7341865/checking-if-jquery-is-loaded-using-javascript) if jQuery get's loaded or not, and it seems to be loaded. 
- If I also add the jQuery file to the Media class, the error messages disappears, but the toggling doesn't work either
- Then I tried to wrap it in jQuery.document.ready but that didn't help either. Do you have an idea left, what the issue might be? 
Thank you a lot in advance!